server/lib/src/idm/ldap.rs

```diff
@@ -643,15 +661,19 @@ mod tests {
         let pce = UnixPasswordChangeEvent::new_internal(UUID_ADMIN, TEST_PASSWORD);
 
         assert!(idms_prox_write.set_unix_account_password(&pce).is_ok());
+        assert!(idms_prox_write
+            .qs_write
+            .set_domain_ldap_allow_unix_pw_bind(true)
+            .is_ok());
         assert!(idms_prox_write.commit().is_ok());
-
         let anon_t = ldaps.do_bind(idms, "", "").await.unwrap().unwrap();
         assert!(anon_t.effective_session == LdapSession::UnixBind(UUID_ANONYMOUS));
         assert!(
             ldaps.do_bind(idms, "", "test").await.unwrap_err() == OperationError::NotAuthenticated
         );
 
         // Now test the admin and various DN's
+        // Writing test for optional use of POSIX password for bind
```

Comment:
Probably should make a new test case here, that also tests when the condition is *false*.